### DISCUSSION TITLE
When playing a map, fps slowly decrease from 240 to 170

### DISCUSSION THREAD

**Discussion by MoutonBinoclard** [ppy/osu#29305] [Q&A] [answered]
### Type

Performance

### Bug description

When i launch a map, i see my fps decreasing as the i play the map, it's very recent, but the problem is I didn't do anything that could cause this.
I'll join my log and a video (check the video at 50sec)
i'll drop the video link here : https://drive.google.com/file/d/1394xeV6CnycUv8tQoHm-1M-ii4GgLieO/view?usp=sharing

I tried to restart, change map, and also deleteing all the game and the temporary file, but it stay the same

### Version

2024.731.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16501407/compressed-logs.zip)

**Comment by MoutonBinoclard** [comment]
Even, when i pause the game, the fps don't increase
![image](https://github.com/user-attachments/assets/05ae167f-d339-4a78-bbe1-9eb392581d63)

**Comment by MoutonBinoclard** [comment]
I also tried to use one of the default skin (argon pro), and changing rendering mode, but i didn't do a thing

**Comment by MoutonBinoclard** [comment]
I also find out, some map are worse than other
this one :
https://osu.ppy.sh/beatmapsets/639700#osu/1356687
cause the bug after only 30 second of playtime

**Reply by peppy** [reply]
Can you take a video with the frame graphs visible on said beatmap?

Ctrl + F11 twice.

**Reply by MoutonBinoclard** [reply]
Ok this is weirder than I though:
The fps drops when the graph arent visible 
See for yourself (and sorry for my accent)
https://drive.google.com/file/d/1yqnvVDHT1j8n1X2JG6LFu2Il2mqEyRud/view?usp=sharing

**Reply by MoutonBinoclard** [reply]
I would really wanted to upload the log, but sadly, there is an error when i tried to export them
![image](https://github.com/user-attachments/assets/4c00a4c0-7529-44cf-8a50-5f85cc7d99ce)

**Reply by MoutonBinoclard** [reply]
![image](https://github.com/user-attachments/assets/aa25721e-2c78-489a-b4f5-12cd319e6999)
![image](https://github.com/user-attachments/assets/3940aedc-a6e8-45b8-b5dd-340ae01b2c92)
here's is the fps graph when i got drops
but when opening it, all the problems dissappear

**Reply by smoogipoo** [reply]
Try going into options, then searching for "renderer", and changing it to OpenGL. It really looks like your GPU is misbehaving somehow - either it's overheating or there's something else it doesn't like.

**Reply by MoutonBinoclard** [reply]
I tried other render method, but it didn't solve the problem, however, I noticed overheating may be a thing, i'll buy some thermal patty and thermal paste and try to fix that

**Comment by smoogipoo** [comment]
It sounds like you have an overheating CPU/GPU, or a background process running.

Nothing obvious in the logs.

**Comment by MoutonBinoclard** [comment, marked as answer]
Ok I find myself, unable to reproduce the bug, but if it comes again, i'll record it.
But i sure at 100% that it wasn't a cpu/gpu limitation, i also look the task manager, and both were under 40% usage (even in game)

**Comment by MoutonBinoclard** [comment]
@smoogipoo @peppy
Find out, the problem isn't from osu! or my computer but from nvidia latest driver
I rolled back a few version and it works perfectly fine again

For those who have the same issue on a 1650ti, try switching back to the driver with this version : 551.86
